regression/cbmc-java/generic_class_bound1/test.desc

```diff
@@ -7,5 +7,5 @@ Gn.class
 .*file Gn.java line 9 function java::Gn.foo1:\(LGn;\)V bytecode-index 1 block 1: FAILED
 .*file Gn.java line 10 function java::Gn.foo1:\(LGn;\)V bytecode-index 4 block 2: FAILED
 .*file Gn.java line 11 function java::Gn.foo1:\(LGn;\)V bytecode-index 5 block 3: FAILED
-.*file Gn.java line 13 function java::Gn.main:\(\[Ljava/lang/String;\)V bytecode-index 2 block 1: SATISFIED
+.*file Gn.java line 13 function java::Gn.main:\(\[Ljava/lang/String;\)V bytecode-index 2 block 2: SATISFIED
```

Comment:
Not especially important but do you know why?

Reply:
Yes, for example on accessing the `System` static class for the first time (e.g. for `System.out.println`) it will call a synthetic `System.clinit` method, and the new function call constitutes a coverage goal. Previously the init of `System.out` et al would have happened during `__CPROVER_initialize`, i.e. at start-of-day, regardless of whether the objects were actually used or not.